index.html

```diff
@@ -293,8 +293,38 @@ <h4><a href="#">LET'S CONNECT </a></h4>
             </ul>
         </div>
     </section>
-</div>
-
-   
+    <!-- This is the footer section of the webpage-->
+    <footer>
+        <section id="contact-form">
+            <h5>Contact me</h5>
+            <p>
+            If you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding.
+            I’d love to help with it
+            </p>
+            <form class="input" action="https://formspree.io/f/xlezvgbv" method="post">
```

Comment:
- [ ] [OPTIONAL] Kindly ensure you follow the contact form styling properly. There is a horizontal overflow currently and a background image on the text area. Let's fix this for a better user experience UX.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:

![Screenshot from 2022-06-08 12-34-12](https://user-images.githubusercontent.com/58692590/172607473-85ccc77e-d346-4036-976f-0eb3baf47a2b.png) | ![Screenshot from 2022-06-08 12-38-00](https://user-images.githubusercontent.com/58692590/172607494-35fe434c-d34a-4c14-816c-85c6e9d82010.png)